### DISCUSSION TITLE
Customize categories display on designer

### DISCUSSION THREAD

**Discussion by Naim-h** [elsa-workflows/elsa-core#3017] [Q&A]
Hi
I need to display the activity categories as tree view ,can you please guild me  how can I implement it? 
![image](https://user-images.githubusercontent.com/60927103/168055655-530e2984-1ce5-472e-9dba-33cb46aa3120.png)

**Comment by sfmskywalker** [comment]
Probably the best way is to fork the project and implement a treeview component that can be reused to replace the current component.
Ideally, you do it in such a way that we have two components:

1. The current one
2. The treeview one.

Then users of the designer can configure which component to use.
